### PR TITLE
set up qunit and make first test

### PR DESCRIPTION
**Pull request by mattlub**

### CODE REVIEW

File: tests/front-end/tests.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+QUnit.test("a test", function(assert){
+  function name( name){
+    return name;
+  }
+  var myname ="hiba";
+  assert.equal(myname,"hiba","the same name");
+});
```
Comment (jwld) [inline tests/front-end/tests.js]: spacing issues in lines 1, 2, 5 and 6

**Review by jwld — CHANGES_REQUESTED** [review]
spacing issuez

File: tests/front-end/tests.js (hunk 1 above)
Comment (mattlub) [reply tests/front-end/tests.js]: bastard

Comment (jwld) [reply tests/front-end/tests.js]: <3

File: tests/front-end/tests.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+QUnit.test("a test", function(assert) {
+  function name(name) {
+    return name;
+  }
```
Comment (jwld) [inline tests/front-end/tests.js]: why the function?

**Review by jwld — CHANGES_REQUESTED** [review]
improve qunit test

**Pull request merged**